metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-tagmail",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Please revert this metadata bump. We prefer to handle that internally when preparing a release.